Helper/Cart.php

```diff
@@ -882,7 +882,10 @@ public function getBoltpayOrder($paymentOnly, $placeOrderPayload = null, $storeI
             $this->deactivateSessionQuote($quote);
             return;
         }
-
+        
+        if (version_compare($this->configHelper->getStoreVersion(), '2.3.6', '<') && $this->deciderHelper->isAPIDrivenIntegrationEnabled()) {
```

Comment:
According to this commit https://github.com/magento/magento2/commit/c5563f254f3e28d9c1b6cc4ae04bcad4fd7faf56 issue happens with Magento >= 2.3.4
Is this true? Should we save only for 2.3.4 and 2.3.5 ?

Reply:
+1 this

Reply:
@vitaliyreznikov @mikeshutov-bolt It impacts all the versions < 2.3.6, cause the core code does not pass the parent item and calls setPrice. The title of that commit only mentions bundle product, but actually this bug also effects configurable product.